### PR TITLE
Add everything!

### PR DESCRIPTION
**Pull request by Avaq**

### CODE REVIEW

**Comment by Avaq** [comment]
At the time of writing, coveralls is down, so we'll have to wait to see if reports are working.

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+});
+
+test ('createReducer', function() {
+  var result = createReducer ('Test') ({foo: mockHandler});
+  eq (typeof result) ('object');
+  eq (typeof result.reducer) ('function');
+  eq (typeof result.types) ('object');
+  eq (result.reducer ({}, {type: 'Warped/Test/foo', payload: 'test'}))
+     ({actionPayload: 'test'});
+  eq (result.types) ({foo: 'Warped/Test/foo'});
```
Comment (paolobueno) [inline test/index.js]: The space before function params really makes it look like a different language!

File: index.mjs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+//. [![Build Status](https://travis-ci.com/wearereasonablepeople/warped-reducers.svg?branch=master)](https://travis-ci.com/wearereasonablepeople/warped-reducers)
+//.
+//. Compile a standard Redux reducer from a brief definition.
+//.
+//. Works nicely with [Warped Components][1].
+//.
+//. Usage in Node depends on `--experimental-modules`.
+//. With older Node versions, use [std/esm][2].
+
+// createType :: (String, String) -> String
```
Comment (paolobueno) [inline index.mjs]: Nice tooling for embedding md

File: test/index.js (hunk 1 above)
Comment (Avaq) [reply test/index.js]: That's the trick. See https://github.com/sanctuary-js/sanctuary/issues/438 :)

File: index.mjs (hunk 2 above)
Comment (Avaq) [reply index.mjs]: That's https://github.com/plaid/transcribe under the hood.

**Comment by Avaq** [comment]
There we go. With Coveralls working, this work should be ready for a merge.

**Pull request merged**